.prettierrc.js

```diff
@@ -8,4 +8,8 @@ module.exports = {
   singleQuote: true,
   tabWidth: 2,
   trailingComma: 'all',
+  plugins: ['prettier-plugin-tailwindcss'],
+  tailwindConfig: './apps/storybook-react/tailwind.config.js', // Main web config
+  tailwindFunctions: ['tw', 'twMerge'], // Support twrnc and twMerge template literals and utility functions
+  tailwindAttributes: ['twClassName'], // Support twClassName prop
```

Comment:
Configures the tailwind css prettier plugin to ensure it works for all combinations within our components. [See docs](https://github.com/tailwindlabs/prettier-plugin-tailwindcss)